lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -136,6 +198,9 @@ def define_let_it_be_methods(identifier, **modifiers)
 
     config.register_modifier :refind do |record, val|
       next record unless val
+
+      TestProf::LetItBe::Stoplist.push(record)
```

Comment:
After giving it another thought, when `refind`/`reload` are in action, this doesn't mean that it's protected from second-order leakages. So it makes sense not to freeze itself, but still walk and freeze its associated records.

Reply:
Let me reconsider this in the light of my recent discovery. I might have to simplify a couple of things.